### ISSUE TITLE
How to implement 5-fold cross validation

### ISSUE THREAD

**Issue by ahmedeqbal** [nibtehaz/MultiResUNet#12]
Hello Nabil,

I check your demo code, i want to to implement 5-fold cross validation in it, and i never found any help anywhere.

Please can you share how i can implement 5-fold cross validation on this code?

Thanks

**Comment by nibtehaz** [comment]
Hello, thank you for your interest in our multiresunet project. I am sorry for my delayed response as I have not been active lately.

To implement the 5 fold cross validation we used the standard scikit learn function, sklearn.model_selection.KFold.

Again I apologize for the delayed response.

**Comment by ahmedeqbal** [comment]
Dear Nabil,

i hope you are doing well.

Thanks for valuable information, it really help me.

i have one more question, To compare image ground truth, how i can draw Blue, Red and Gree line marker, which tool or script is used for this?

![GTpng](https://user-images.githubusercontent.com/7148894/102374544-47d1f400-3fe3-11eb-9c70-4d41c86ea0f3.png)

i would really appreciate that if you can help in this regard.

**Comment by nibtehaz** [comment]
Sorry, I missed this comment. I just compute the edges of the segmentation mask and color them.
